nodejs/BUILD

```diff
@@ -27,7 +27,10 @@ NODEJS_MAJOR_VERISONS = ("12", "14", "16")
 [
     container_test(
         name = "nodejs" + major_version + ("" if (not mode) else mode) + "_" + arch + "_" + distro + "_test",
-        configs = ["testdata/nodejs" + major_version + ".yaml"],
+        configs = [
+            "testdata/nodejs" + major_version + ".yaml",
+            "testdata/check_npm.yaml"
```

Comment:
add a `,` at the end to appease buildifier

Reply:
Added trailing comma